client/jetpack-cloud/sections/partner-portal/license-list/empty-list.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement, useContext } from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import { getLicenseCounts } from 'calypso/state/partner-portal/licenses/selectors';
+import LicenseListContext from 'calypso/jetpack-cloud/sections/partner-portal/license-list-context';
+import { LicenseFilter } from 'calypso/jetpack-cloud/sections/partner-portal/types';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function LicenseListEmpty(): ReactElement {
+	const translate = useTranslate();
+	const { filter } = useContext( LicenseListContext );
+	const counts = useSelector( getLicenseCounts );
+	const hasAssignedLicenses = counts[ LicenseFilter.Attached ] > 0;
+
+	const licenseFilterStatusMap = {
+		[ LicenseFilter.NotRevoked ]: translate( 'active' ),
+		[ LicenseFilter.Attached ]: translate( 'assigned' ),
+		[ LicenseFilter.Detached ]: translate( 'unassigned' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Unassigned' )` *ES Score: 15*